toolchain.txt

```diff
@@ -16,18 +16,10 @@ strip = '/tools/riscv/bin/riscv32-unknown-elf-strip'
 [properties]
 has_function_printf = false
 c_args = [
-  '-fvisibility=hidden',
-  '-g',
   '-mabi=ilp32',
   '-march=rv32imc',
   '-mcmodel=medany',
-  '-nostartfiles',
-  '-nostdlib',
-  '-Os',
-  '-static',
-  '-Wall',
-  '-Werror']
```

Comment:
Where did those two go?

Reply:
They're already in `add_project_arguments` above my changes in meson.build

Reply:
Ah, I see. Can we merge the `add_project_arguments` blocks then?

Reply:
We could, but the `add_project_arguments` above are duplicated for `native: true` and `native: false` (cross-product with c, cpp), whereas these two `add_project_arguments` are for `native: false` only (cross-product with c, cpp), which is why I wanted to keep them separate.